scripts/index_incipits.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+'''
+Skip to index PEMM incipits in Solr for search.
+
+Requires parasolr:
+
+    pip install parasolr
+
+Create a solr core with the local solr configset for
+searching on Geʽez / Fidäl:
+
+    solr create_core -c pemm_incipit -d solr_conf
+
+Run this script, providing solr connection details and a path
+to the PEMM project CSV file for story instances.
+
+    python index_incipits.py http://localhost:8983/solr/ pemm_incipit \
+        /path/to/data/story_instance.csv
+
+'''
+
+import argparse
+import csv
+
+from parasolr.solr.client import SolrClient
+
+
+def index_incipits(solr_url, solr_core, incipitfile):
```

Comment:
depending on the context in which this will run (but at least for trying it locally) logging would be good to have here. progress bar could be cool but I'm guessing it's actually so fast that we don't even need one?

Reply:
Right now the complete run is very quick. What output would be useful? Number of records indexed?

Reply:
yep, I think that's the main thing.